### DISCUSSION TITLE
我的人生管理系统 · BMPI

### DISCUSSION THREAD

**Discussion by giscus** [bmpi-dev/bmpi.dev#140] [Announcements]
# 我的人生管理系统 · BMPI

本文分享我使用纯文本管理我的目标(OKR)、时间(GTD)、笔记(Logseq)与财务(Beancount)的人生管理经验。

https://www.bmpi.dev/self/life-in-plain-text/

**Comment by codefuturedalao** [comment]
请问您的字体设置是什么，可以分享一下么？

**Comment by codefuturedalao** [comment]
您的邮件我已收到

**Comment by Norman-h** [comment]
您好，我想请问下您是如何导出 logseq 的文件的，logseq 的 md 文件全部是无序列表的形式，这样看起来非常的难受呀
